assets/UE4SS-settings.ini

```diff
@@ -87,6 +87,7 @@ MakeAllConfigsEngineConfig = 1
 ConsoleEnabled = 1
 GuiConsoleEnabled = 1
 GuiConsoleVisible = 1
+FullMemoryDump = 0
```

Comment:
I don't really like this being grouped.  At this point the live view debugger is more of a mod debugger than a UE4SS debugger, and I think the 2 concepts should be distinguished.